grails-app/views/header/_mainPortalHeader.gsp

```diff
@@ -1,13 +1,17 @@
 <div id="header">
 
     <div id="logoContainer">
-        <a href="landing"><img src="${portalBranding.logoImage}" alt="logo" id="logo" />
+        <a href="landing"><img src="${portalBranding.logoImage}" alt="main logo" id="logo" height="80" />
         </a>
     </div>
     <div id="landingHeaderContainer" class="headerHeightOverlord">
         <h1 id="headerTitle">${portalBranding.siteHeader}</h1>
     </div>
-
+    <g:if test="${portalBranding.secondaryLogoImage}">
```

Comment:
@danfruehauf Sorry if `secondaryLogoImage` is an empty string the user will see the text from the alt attribute. The image tag should not be rendered. The test in the gsp will need to stay. 
There is no such test on the main `logoImage` because it is expected to exist.